modules/serverless-logic/pages/release-notes.adoc

```diff
@@ -3,13 +3,13 @@
 
 == Notable changes
 
-* link:https://issues.redhat.com/browse/SRVLOGIC-7[SRVLOGIC-7] - [core] Simplified serverless knative function invocation
-* link:https://issues.redhat.com/browse/SRVLOGIC-47[SRVLOGIC-47] - [use case] Improve Workflow Timeouts support (community only)
 * link:https://issues.redhat.com/browse/SRVLOGIC-62[SRVLOGIC-62] - Data Index as Quarkus extension
-* link:https://issues.redhat.com/browse/SRVLOGIC-65[SRVLOGIC-65] - Upgrade Workflow Runtime to RHBQ 2.13
-* link:https://issues.redhat.com/browse/SRVLOGIC-66[SRVLOGIC-66] - [SWF Editor] Add support to Async API
-* link:https://issues.redhat.com/browse/SRVLOGIC-67[SRVLOGIC-67] - Serverless Workflow Visualization Improvements - DP5
-* link:https://issues.redhat.com/browse/SRVLOGIC-69[SRVLOGIC-69] - [SWF Dev UI] Add support to start workflows using gateway API and cloud events
-* link:https://issues.redhat.com/browse/SRVLOGIC-76[SRVLOGIC-76] - Data-index prod ephemeral image is missing the `KOGITO_DATA_INDEX_QUARKUS_PROFILE`
+* link:https://issues.redhat.com/browse/SRVLOGIC-136[SRVLOGIC-136] - Recollection of DP6 small features
+* link:https://issues.redhat.com/browse/SRVLOGIC-130[SRVLOGIC-130] - Execute serverless workflow without code generation
+* link:https://issues.redhat.com/browse/SRVLOGIC-129[SRVLOGIC-129] - Jobs Service features, fixes, and enhancements available in the technical preview
+* link:https://issues.redhat.com/browse/SRVLOGIC-132[SRVLOGIC-132] - Jobs Service Embedded Addon
```

Comment:
SRVLOGIC-129, SRVLOGIC-130, SRVLOGIC-132 were not a part of DP6.
```suggestion
* link:https://issues.redhat.com/browse/SRVLOGIC-68[SRVLOGIC-68] - YAML Serverless Workflow parsing on Stunner
* link:https://issues.redhat.com/browse/SRVLOGIC-133[SRVLOGIC-133] - Support validation of output schema
* link:https://issues.redhat.com/browse/SRVLOGIC-136[SRVLOGIC-136] - Recollection of DP6 small features
* link:https://issues.redhat.com/browse/SRVLOGIC-145[SRVLOGIC-145] - Serverless Workflow Visualization Improvements - DP6
```

Reply:
@MarianMacik Some of the jiras under SRVLOGIC-62,SRVLOGIC-129, SRVLOGIC-130, SRVLOGIC-132  were released in DP6. I got them from the release notes of 1.36,1.37 & 1.38. Please suggest if I need to remove them or keep in  this case.